src/components/AdminFields/Organizations/OrganizationPermissions/OrganizationEditor.js

```diff
@@ -127,7 +127,8 @@ class OrganizationEditor extends React.Component {
             const {data} = (await client.get('organization_class'))?.data;
             // make Muu yhteisö class = org:10 last choice
             const otherClass = data.find((org) => org.id === 'org:10'),
-                rest = data.filter((org) => org.id !== 'org:10');
+                rest = data.filter((org) => org.id !== 'org:10')
+                    .sort((prev, next)=> (prev.name).localeCompare(next.name));
```

Comment:
This does not fix the reported issue where Säätiö and Seurakunta are in the wrong order. Function `localeCompare` might assume letter ä equals to a and therefore fails to sort these words. The following code seems to work in FireFox:
`.sort((prev, next)=> prev.name > next.name);`
Neither solution seems to work in Chrome

Reply:
adding locale `fi` param to the original solution seems to work: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#using_locales

Reply:
yes, locale was missing that made it not to work on the original solution.